packages/braintree-integration/src/BraintreeAcceleratedCheckout/BraintreeAcceleratedCheckoutPaymentMethod.tsx

```diff
@@ -11,9 +11,11 @@ import {
 import { FormContext, LoadingOverlay } from '@bigcommerce/checkout/ui';
 
 import BraintreeAcceleratedCheckoutForm from './components/BraintreeAcceleratedCheckoutForm';
+import { CardInstrument } from '@bigcommerce/checkout-sdk';
```

Comment:
This line should throw eslint warning, because the import should be in another import group. Please, fix it